scriptHelper.js

```diff
@@ -1,43 +1,96 @@
-// Write your helper functions here!
 require('isomorphic-fetch');
-
+    
 function addDestinationInfo(document, name, diameter, star, distance, moons, imageUrl) {
-   // Here is the HTML formatting for our mission target div.
-   /*
-                <h2>Mission Destination</h2>
-                <ol>
-                    <li>Name: </li>
-                    <li>Diameter: </li>
-                    <li>Star: ${star}</li>
-                    <li>Distance from Earth: </li>
-                    <li>Number of Moons: </li>
-                </ol>
-                <img src="">
-   */
+    // Here is the HTML formatting for our mission target div.
+    let missionTarget=document.getElementById("missionTarget")
+    missionTarget.innerHTML=`
+                    <h2>Mission Destination</h2>
+                    <ol>
+                        <li>Name: ${name} </li>
+                        <li>Diameter:${diameter} </li>
+                        <li>Star: ${star}</li>
+                        <li>Distance from Earth:${distance} </li>
+                        <li>Number of Moons: ${moons}</li>
+                    </ol>
+                    <img src="${imageUrl}">`
 }
 
 function validateInput(testInput) {
-   
+    if(testInput===""){
+        return "Empty"
+    } else if (isNaN(Number(testInput))){
+        return "Not a Number"
+    } else {
+        return "Is a Number"
```

Comment:
```suggestion
        return "Empty";
    } else if (isNaN(Number(testInput))){
        return "Not a Number";
    } else {
        return "Is a Number";
```

Reply:
Same thing about semi-colons being best practice. On line 21, you do not need to use Number() in this case to convert the input into a number before using it in `isNaN`.  Coercion is the conversion of a value into another of a different data type. There may be use cases where `isNaN` may need to be used in another way. Check out the link below for an example:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#confusing_special-case_behavior